cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -191,3 +191,303 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
 
     # Revert back to original config
     reconfigure_vm(small_vm, orig_config)
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.parametrize('change_type', ['sockets', 'memory'])
+def test_vm_reconfig_add_remove_hw_hot_vmware():
```

Comment:
@niyazRedhat: `travis` is failing due to this.... you are trying to parametrize manual test but `change_type` not used.

```
def test_vm_reconfig_add_remove_hw_hot_vmware(change_type):
```